src/backend/phpstan.neon

```diff
@@ -7,7 +7,7 @@ parameters:
         - app/
 
     # Level 9 is the highest level
-    level: 1
+    level: 2
```

Comment:
We can't do this yet, as not all level 2 errors will be resolved just yet. This will make our CI/CD fail.